Ntd branch

ntd6039 opened the pull request:
## Title: Create Calendar Front end 



## Description 

Added the calendar.tsx, calendar-days.css, calendar-days.js
- For frontend work, you _must_ include screenshots of your work in the description.

need Help because my arrows are don't look like arrows. Not sure if this slays or not. meow 

## Types of changes

![Screenshot 2023-03-30 at 11 31 36 PM](https://user-images.githubusercontent.com/97009566/229015956-7d14a453-49ef-4de3-bee0-f80df5eb7c83.png)

What types of changes does your code introduce to WeWrap?
_Put an `x` in the boxes that apply_
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have made sure that eslint was passed locally
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

ntd6039 committed:
Add calendar to app.tsx

ntd6039 committed:
Add calendar component

kdata12 commented on frontend/src/components/calendar_frontend/calendar-days.js:
```diff
@@ -0,0 +1,43 @@
+    currentDays.push(calendarDay);
+  }
+
+  return (
```
Since this function returns html element with javascript using `JSX`, it would help to rename this file to `calendar-days.jsx` to help other people know this file contains both HTML-like code and JavaScript logic

kdata12 commented on frontend/src/components/calendar_frontend/calendar.tsx:
```diff
@@ -0,0 +1,55 @@
+        </div>
+        <div className="tools">
+          <button onClick={previousDay}>
+            <span className="material-icons">
```
you forgot add CSS for this span tag? Is that the reason why your arrow doesn't look like arrow?

kdata12 commented on frontend/src/components/calendar_frontend/calendar.tsx:
```diff
@@ -0,0 +1,55 @@
+          </button>
+          <p>{months[currentDay.getMonth()].substring(0, 3)} {currentDay.getDate()}</p>
+          <button onClick={nextDay}>
+            <span className="material-icons">
```
same here, didn't add css?

ntd6039 committed:
Added Today button and CSS stylings

ntd6039 reviewed (commented):
Added in today button, styled with CSS.

ntd6039 committed:
Merge remote-tracking branch 'origin' into Ntd-Branch-

ntd6039 commented:
![Screenshot 2023-05-12 at 11 33 16 PM](https://github.com/wewrap/togetherwewrap/assets/97009566/62a27eb3-8566-4f36-8ea6-8a4469d97323)


updated and styled, added button
